spec/acts_as_revisionable_spec.rb

```diff
@@ -75,7 +75,9 @@ class RevisionableTestModel < ActiveRecord::Base
 
       attr_protected :secret
```

Comment:
@mhoofman Yes, but it's available as an external gem now. If you run the tests with "appraisal" it'll run with the custom gemfile.